### PR TITLE
add link and styling

### PR DESCRIPTION
**Pull request by NoelleDL**
### What

Create "Skip to main content" link

### How to review

First tab on page will make "Skip to main content" link appear above the nav bar.

### CODE REVIEW

File: CMS/static/scss/global.scss [hunk 1]
```diff
@@ -218,3 +218,19 @@ a[target=_blank] {
...
 
+a.skip-main {
+  position: absolute;
+  left: -999px;
+  width: 1px;
+  height: 1px;
+  top: auto;
+}
+
+a.skip-main:focus {
```
Comment (VivianAllen) [inline CMS/static/scss/global.scss:229]: could make use of scss nesting and the '&' thing here?

Comment (NoelleDL) [reply CMS/static/scss/global.scss]: Did it. Thank you!

Comment (NoelleDL) [reply CMS/static/scss/global.scss]: Could you give me a thumbs up if it's good to go? @VivianAllen

Comment (mwnicks) [reply CMS/static/scss/global.scss]: 👍 from me

**Pull request merged**